### ISSUE TITLE
hyper terminal does not display icons in the terminal prompt.

### ISSUE THREAD

**Issue by adarsha-jll** [vercel/hyper#7010]
- [ ] Your Hyper.app version is **3.4.1**. Please verify you're using the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate
---
- **Any relevant information from devtools?** _(CMD+OPTION+I on macOS, CTRL+SHIFT+I elsewhere)_:

- **Is the issue reproducible in vanilla Hyper.app?**

## Issue
hyper terminal does not display icons in the terminal prompt.
i have installed powerlevel10k, and necessary fonts.

<img width="1792" alt="Screenshot 2023-02-05 at 12 46 40 PM" src="https://user-images.githubusercontent.com/107102139/216806702-e93eedf7-1cff-4ea4-94f0-80d81af7e7e2.png">

---

- **Hyper version**: undefined "3.4.1"
- **OS ARCH VERSION:** darwin x64 22.3.0
- **Electron:** 20.3.6  **LANG:** undefined
- **SHELL:** /bin/zsh   **TERM:** undefined
<details><summary><strong>.hyper.js contents</strong></summary>

```json
{
  "updateChannel": "stable",
  "fontSize": 12,
  "fontFamily": "\"Hack Nerd\", Hack, Menlo, \"DejaVu Sans Mono\", Consolas, \"Lucida Console\", monospace",
  "fontWeight": "normal",
  "fontWeightBold": "bold",
  "lineHeight": 1,
  "letterSpacing": 0,
  "cursorColor": "rgba(248,28,229,0.8)",
  "cursorAccentColor": "#000",
  "cursorShape": "BLOCK",
  "cursorBlink": false,
  "foregroundColor": "#fff",
  "backgroundColor": "#000",
  "selectionColor": "rgba(248,28,229,0.3)",
  "borderColor": "#333",
  "css": "",
  "termCSS": "",
  "workingDirectory": "",
  "showHamburgerMenu": "",
  "showWindowControls": "",
  "padding": "12px 14px",
  "colors": {
    "black": "#000000",
    "red": "#C51E14",
    "green": "#1DC121",
    "yellow": "#C7C329",
    "blue": "#0A2FC4",
    "magenta": "#C839C5",
    "cyan": "#20C5C6",
    "white": "#C7C7C7",
    "lightBlack": "#686868",
    "lightRed": "#FD6F6B",
    "lightGreen": "#67F86F",
    "lightYellow": "#FFFA72",
    "lightBlue": "#6A76FB",
    "lightMagenta": "#FD7CFC",
    "lightCyan": "#68FDFE",
    "lightWhite": "#FFFFFF",
    "limeGreen": "#32CD32",
    "lightCoral": "#F08080"
  },
  "shell": "",
  "shellArgs": [
    "--login"
  ],
  "env": {},
  "bell": "SOUND",
  "copyOnSelect": false,
  "defaultSSHApp": true,
  "quickEdit": false,
  "macOptionSelectionMode": "vertical",
  "webGLRenderer": true,
  "webLinksActivationKey": "",
  "disableLigatures": true,
  "disableAutoUpdates": false,
  "screenReaderMode": false,
  "preserveCWD": true
}
```
</details>
<details><summary><strong>plugins</strong></summary>

```json
{
  "plugins": [
    "hyper-search",
    "shades-of-purple-hyper"
  ],
  "localPlugins": []
}
```
</details>

**Comment by ohhheyyyy** [comment]
Hi @adarsha-jll! What happens when you edit the `fontFamily` key / value pair in your `.hyper.js` file to:
`fontFamily: '"Hack Nerd Font", monospace',`?

I installed a fresh version of Hyper and then added fonts using [homebrew](https://brew.sh/) with the following commands:
```
brew tap homebrew/cask-fonts
brew install font-hack-nerd-font
```

Thereafter, I amended the `fontFamily` key / value pair in my `.hyper.js` file such that it reads:
`fontFamily: '"Hack Nerd Font", monospace',`

If you tried these suggestions, has the appearance of your command line prompt changed to display icons as expected?
